requirements.txt

```diff
@@ -25,4 +25,4 @@ pydantic
 pytest-cov
 plotly
 tqdm
-git+https://github.com/SheffieldSolar/PV_Live-API
+pvlive_api @ git+https://github.com/SheffieldSolar/PV_Live-API.git@master#egg=pvlive_api
```

Comment:
Not sure if you've had some trouble with this, but https://github.com/openclimatefix/nowcasting_dataset/issues/137 talks about how because of the git dependency, we can't upload to PyPi currently.

Reply:
Hmm, maybe we make this an optional requirement, and that would help things

Reply:
Yeah, PyPi still unfortunately still won't accept it. So maybe we can make it a separate install step? But in any case, this shouldn't hold up this PR, its a different problem anyway